src/passwordless/client.py

```diff
@@ -0,0 +1,223 @@
+        self.options = options
+        self.session = session
+
+    def create_alias(self, create_alias: CreateAlias) -> None:
+        schema = CreateAliasSchema()
+        request_data = schema.dumps(create_alias)
+
+        request = self.post_request('/alias', request_data)
+
+        self.send_request(request)
+
+    def get_aliases(self, user_id: str) -> List[Alias]:
+        request = self.get_request('/alias/list', {"userId": user_id})
+        response_data = self.send_request(request)
+
+        schema = AliasListResponseSchema()
+        list_response: ListResponse = schema.loads(response_data)
+
+        return list_response.values
+
+    def update_apps_feature(self, update_apps_feature: UpdateAppsFeature) -> None:
+        schema = UpdateAppsFeatureSchema()
+        request_data = schema.dumps(update_apps_feature)
+
+        request = self.post_request('/apps/features', request_data)
+
+        self.send_request(request)
+
+    def delete_credential(self, delete_credential: DeleteCredential) -> None:
+        schema = DeleteCredentialSchema()
+        request_data = schema.dumps(delete_credential)
+
+        request = self.post_request('/credentials/delete', request_data)
+
+        self.send_request(request)
+
+    def get_credentials(self, user_id: str) -> List[Credential]:
+        request = self.get_request('/credentials/list', {"userId": user_id})
+        response_data = self.send_request(request)
+
+        schema = CredentialListResponseSchema()
+        list_response: ListResponse = schema.loads(response_data)
+
+        return list_response.values
+
+    def register_token(self, register_token: RegisterToken) -> RegisteredToken:
+        request_schema = RegisterTokenSchema()
+        request_data = request_schema.dumps(register_token)
+
+        request = self.post_request('/register/token', request_data)
+
+        response_data = self.send_request(request)
+
+        response_schema = RegisteredTokenSchema()
+        return response_schema.loads(response_data)
+
+    def sign_in(self, verify_sign_in: VerifySignIn) -> VerifiedUser:
+        request_schema = VerifySignInSchema()
+        request_data = request_schema.dumps(verify_sign_in)
+
+        request = self.post_request('/signin/verify', request_data)
+
+        response_data = self.send_request(request)
+
+        response_schema = VerifiedUserSchema()
+        return response_schema.loads(response_data)
+
+    def get_users(self) -> List[UserSummary]:
+        request = self.get_request('/users/list')
+        response_data = self.send_request(request)
+
+        schema = UserSummaryListResponseSchema()
+        list_response: ListResponse = schema.loads(response_data)
+
+        return list_response.values
+
+    def delete_user(self, delete_user: DeleteUser) -> None:
+        schema = DeleteUserSchema()
+        request_data = schema.dumps(delete_user)
+
+        request = self.post_request('/users/delete', request_data)
+
+        self.send_request(request)
```

Comment:
I wonder whether should move those private methods out of this class or even python file.